### PR TITLE
reset alarm manager after reboot

### PR DESCRIPTION
**Pull request by gbielanski**
please review

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -32,8 +34,13 @@
...
-
+        <receiver android:name=".AlarmReceiver">
+            <intent-filter>
+                <action android:name="android.intent.action.BOOT_COMPLETED" />
+                <action android:name="android.intent.action.QUICKBOOT_POWERON" />
 
+                <category android:name="android.intent.category.DEFAULT" />
+            </intent-filter>
+        </receiver>
+    </application>
```
Comment (miltomasz) [inline app/src/main/AndroidManifest.xml:46]: Format code above

File: app/src/main/java/pl/example/android/garbageapp/AlarmReceiver.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+import android.content.Context;
+import android.content.Intent;
+import android.util.Log;
+
+import pl.example.android.garbageapp.utils.NotificationUtils;
+
+public class AlarmReceiver extends BroadcastReceiver {
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        Log.d("AlarmReceiver", "onReceive");
```
Comment (miltomasz) [inline app/src/main/java/pl/example/android/garbageapp/AlarmReceiver.java]: Create tag as final string constant (AlarmReceiver.class.getSimpleName();)

**Review by miltomasz — CHANGES_REQUESTED** [review]
Just correct formatting and TAG.

**Comment by gbielanski** [comment]
Please review

**Review by miltomasz — APPROVED** [review]

**Pull request merged**